src/components/BookList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const BookList = () => {
+  const books = [
+    {
+      id: 1,
+      title: 'go',
+      author: 'out',
+    },
+    {
+      id: 2,
+      title: 'go',
+      author: 'out',
+    },
+    {
+      id: 3,
+      title: 'go',
+      author: 'out',
+    },
+  ];
+  return (
+    <ul>
+      {books.map((book) => (
+        <li key={book.id}>
+          {book.title}
+          {' '}
+          {book.author}
+        </li>

```

Comment:
- [ ] You have successfully rendered the title and author of the book as the requirements demand. This is great! :+1: . The same requirements request us to later add a button for removing books even though the button won't remove the book yet, this would just be a placeholder for now. Would you consider adding the button? , this way, you will move a step towards approval?.